integrations/spark/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,9 @@
+Manifest-Version: 1.0
+Premain-Class: marquez.spark.agent.MarquezAgent
+Agent-Class: marquez.spark.agent.MarquezAgent
+Created-By: Gradle 6.8.3
+Built-By: collado
+Build-Jdk: 15.0.1
+Implementation-Title: spark
+Implementation-Version: 0.13.2-SNAPSHOT

```

Comment:
Should this `MANIFEST.MF` be part of the PR?

Reply:
Nope :)